sourcemash/frontend/assets.py

```diff
@@ -39,8 +39,10 @@
                 "js/views/feed_view.js",
                 "js/views/saved_view.js",
                 "js/views/browse_view.js",
+                "js/views/register_view.js",
```

Comment:
I thought we ended that conversation with login being in the sidenav and making a register view so they can be side-by-side. And the login is always in the sidenav while you're not logged it, so then you'll see the same form twice on the screen: once in the modal and once in the sidenav. That's weird, no?